packages/node/src/exec/default-attenuator.js

```diff
@@ -69,6 +69,16 @@ export const makeGlobalsAttenuator = (
     }
   }
 
+  if (typeof scuttleGlobalThis === 'boolean') {
+    scuttleGlobalThis = {enabled: scuttleGlobalThis}
+  }
+  if (scuttleGlobalThis?.enabled) {
+    if (!Array.isArray(scuttleGlobalThis?.exceptions)) {
+      scuttleGlobalThis.exceptions = []
+    }
+    scuttleGlobalThis.exceptions.push('Map', 'parseFloat', 'WeakMap', 'WeakSet', 'Boolean', 'Number', 'Promise', 'String', 'isNaN')
```

Comment:
pls store this list in `constants.js` with an explanation of what it is and why it exists, e.g.

```js
/**
 * describe me
 */
export const SCUTTLE_EXCEPTIONS = Object.freeze(/** @type {const} */(
  'Map', 'parseFloat', 'WeakMap', 'WeakSet', 'Boolean', 'Number', 'Promise',
  'String', 'isNaN'))
```

Reply:
fixed 93d27f4c7